samtranslator/plugins/globals/globals.py

```diff
@@ -88,7 +88,7 @@ class Globals(object):
     }
     # unreleased_properties *must be* part of supported_properties too
```

Comment:
So it doesn't automatically add `unreleased_properties` to `supported_properties`?

Reply:
right now it doesn't

Reply:
we should remove this manual work